esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -1539,6 +1539,20 @@ int EspHttpBinding::onGetConfig(IEspContext &context, CHttpRequest* request, CHt
     ISecUser* user = context.queryUser();
     if (m_viewConfig || (user && (user->getStatus()==SecUserStatus_Inhouse)))
     {
+        if (getESPContainer() && getESPContainer()->queryApplicationConfig())
```

Comment:
This is primarily used for backward compatibility so that when a user asks to see the config I can give them the correct information.  ESP used to just return the xml file, but the information is generated now so I need to return it.  It will definitely help when testing the new application mode to see how the config differs from what it would have been via legacy configmgr.